tests/e2e/provider_operation_google_docs_cases.py

```
@@ -54,6 +78,202 @@ async def _read_content_outcome(emulate_url: str, preview: dict) -> None:
+        ("POST", update_path, _batch_update_response("2")),
+        ("GET", document_path, _table_document("2", populated=False)),
+        ("POST", update_path, _batch_update_response("3")),
+        ("GET", document_path, _table_document("3", populated=True)),
+        ("POST", update_path, _batch_update_response("4")),
+        ("GET", document_path, _table_document("5", populated=True)),
+    ]
+    for method, path, payload in responses:
+        _arm_json_response(proxy, profile, method, path, payload)
+
+
+def _table_drift_outcome(expected_stage: str, expected_revision: str, failure: str):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_table_drift_outcome`

(ANN202)